File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/non-plagiarized/11/T05.py
import sys

def reverse(number):
    temp = number
    count = 0
    while number != 0:
        number = number // 10
        count += 1
    number = temp
    total = 0
    for i in range(count, 0, -1):
        multiplier = 1
        last_digit = number % 10
        for j in range(1, i):
            multiplier *= 10
        total += last_digit * multiplier
        number //= 10
    print(total)

if __name__ == "__main__":
    number = int(input("Enter an integer: "))
    reverse(number)

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/non-plagiarized/11/T05.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function reverse(number: number): void {
  let temp = number;
  let digits = 0;
  while (number !== 0) {
    number = Math.floor(number / 10);
    digits++;
  }
  number = temp;
  let sum = 0;
  for (let i = digits; i > 0; i--) {
    let multiplier = 1;
    let lastDigit = number % 10;
    for (let j = 1; j < i; j++) {
      multiplier *= 10;
    }
    sum += lastDigit * multiplier;
    number = Math.floor(number / 10);
  }
  console.log(sum);
}

rl.question('Enter an integer: ', (input) => {
  const angka = parseInt(input);
  reverse(angka);
  rl.close();
});

